class ObjectsAlreadyExists(Exception):
    pass


class ValidationError(Exception):
    pass


class UserExists(Exception):
    pass